client/components/forms/form-text-input/test/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import chai, { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { spy } from 'sinon';
+import chaienzyme from 'chai-enzyme';
+import FormTextInput from 'components/forms/form-text-input';
+
+chai.use( chaienzyme() );
+
+describe( '<FormTextInput />', () => {
+	describe( 'rendering', () => {
```

Comment:
I dunno that this `describe` block adds much value, especially when it's the only one.